src/components/data/TableHead.tsx

```diff
@@ -37,6 +37,9 @@ export default function TableHead({
         {...htmlAttributes}
         ref={downcastRef(elementRef)}
         className={classnames(
+          // This ensures the header is drawn on top of positioned content
+          // in table cells.
+          'z-10',
```

Comment:
Do we need it to be that high? Maybe `z-1` is enough for the purpose of this PR while introducing less potential side effects?

Reply:
`z-10` is the lowest "standard value" according to https://tailwindcss.com/docs/z-index, although `z-[1]` should in principle work.

Reply:
Oh! Yeah, I remember seeing my IDE auto-complete `x-1`, that's why I suggested it. I wonder why is it not documented 😅